Add guides (#67)
NHSDigital/prevention-services-design-history: app/guide/how-to-publish.md

---
layout: page
title: How to publish a design history entry
---

> Publishing a design history entry currently requires a bit of familiarity with [GitHub](https://github.com) and [Markdown](https://www.markdownguide.org), but we can help you if needed.

Reviewer: ```suggestion
To publish a design history post, you need to use [GitHub](https://github.com) and [Markdown](https://www.markdownguide.org), but we can help you.
```

To contribute you will need a free GitHub account, and for your account to be added to this project.

> Once this is done, you should follow these steps:

Reviewer: ```suggestion
Once this is done, these steps:
```

1. Create a branch. The name of the branch doesn’t matter too much but you could name it after your post title.

> 2. Add a new markdown file in `app/posts/[service-name]/YYYY/MM/[post-name].md`. It's often easier to duplicate an existing post as that will have the headers you need.

Reviewer: ```suggestion
2. Add a new markdown file in `app/posts/[service-name]/YYYY/MM/[post-name].md`. 

It’s often easier to duplicate an existing post. That way, you will have the headers that contain basic information that all posts need.
```

Author: I think this would break the numbered list and cause it to restart from 1?

Could not use a numbered list at all, and instead just have paragraphs?

Reviewer: or just one paragraph
